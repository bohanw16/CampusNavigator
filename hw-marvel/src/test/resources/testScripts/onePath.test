# Find the only path between two nodes

LoadGraph onePath onePath.csv

FindPath onePath n1 n3