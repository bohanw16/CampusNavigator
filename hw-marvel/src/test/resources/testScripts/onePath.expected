# Find the only path between two nodes

loaded graph onePath

path from n1 to n3:
n1 to n2 via a1
n2 to n3 via a2